### ISSUE TITLE
There is severe flickering on Alex's interface.

### ISSUE THREAD

**Issue by owenzhao** [DanielEdrisian/AlexSideBar-Public#54]
**Describe the bug**
Sometimes, after Alex's interface jumps intensely after a question is asked, it pauses in the middle and I need to manually scroll down to see the results. This seems like an issue with the scrolling calculations of the scrollbar.

**What version are you on currently?**
 - Version 1.3.12 (93)

**Comment by DanielEdrisian** [comment]
Yes this is unfortunately an issue with LazyStack in ScrollView, with varying height children. I haven't been able to find a fix to this, and would probably have to rewrite the whole scrollview in UIKit (unless there's a hack out there that I don't know of).

**Comment by davuthdv** [comment]
I also want to add that scrolling up and down is quite lagged.

This is one of those "tiny annoying stuff" that happens all the time. It's not that big of a deal in itself but because I have to scroll up to click the apply button for every single command, everything just adds up.

Sorry! I also don't have any good suggestion to improve the performance.

**Comment by rudrankriyam** [comment]
> because I have to scroll up to click the apply button for every single command, everything just adds up.
Yeah, a workaround for now is to have sticky headers as mentioned in #49

**Comment by davuthdv** [comment]
A button to "apply all" at the bottom of the chat is also welcomed. I wouldn't need to scroll up.

**Comment by rudrankriyam** [comment]
I agree too; it would give a feeling like "accept all" like in other IDEs

Regards,
Rudrank Riyam <https://www.linkedin.com/in/rudrank/>

**Comment by rudrankriyam** [comment]
I agree too; it would give a feeling like "accept all" like in other IDEs
